llvm/lib/Transforms/IPO/FunctionAttrs.cpp

```diff
@@ -1296,6 +1296,12 @@ static void addNoUndefAttrs(const SCCNodeSet &SCCNodes,
     if (!F->hasExactDefinition())
       return;
```

Comment:
Unrelated to this PR, but I think this should be continue rather than return as well.